### ISSUE TITLE
Rebranding for Beta release.

### ISSUE THREAD

**Issue by t-mullen** [multihack/multihack-web#37]
- [ ] New name. So far "**Relay**" is the best idea.
- [ ] Colour palette and UI consistency.
- [x] Logo. (Thanks @tweakimp)
- [ ] Tagline, product description, important feature list, etc.
- [ ] Documentation improvements.

**Comment by tweakimp** [comment]
Where do you want to go?
 **Do you want it to be the ultimate pair programming tool or to just do what it does now: synchronized coding?**
- I like the name, what do you not like about it?
My suggestion would be "multicode" or "codesync". 
Do you want descriptive name or something abstract?
- Color UI and consistency: what is wrong with the current one?
I can only think of a different code theme and better site design.
- Logo: see 1) and: what do you want here? Im happy with it.

**Comment by t-mullen** [comment]
- The name is more of a subjective thing. Personally I don't find it very easy to say. Also, "Hack" means "to code quickly or for fun" in many circles, but also has negative connotations of illegal activity or poor code quality to others. A "hacker" is a synonym for a cybercriminal in most people's vocabulary. A few people I've talked to have liked the name "Relay", and I think I might go with it if no other options are suggested here.
- The palette is again a matter of taste. I'd like to talk to some people and create a more appealing palette (although the current one isn't too bad). Don't worry, when I change this, I'll also add a theming option so you can revert back to the old one.
- UI consistency mean staying consistent with what people expect in an editor. The web editor has create/delete buttons in odd places, and a few other unconventional things that might confuse users.
- As for the logo, I'll probably go with what you've made. It's really nice and expresses pretty plainly what the product does. :)

This is just an issue for changes to make Multihack... or "Relay"? ... to be more aligned with what people expect from consumer software. Hoping to involve a few designers to improve the quality here.

Nothing's wrong, I just believe there can always be improvement.

**Comment by t-mullen** [comment]
As for your first question, I guess I'll explain where I envision this project going:

Currently there are no satisfactory options for programmer-oriented realtime collaboration. Every solution only allows *one file* within a web editor (dozens of these), uses a simple filesystem sync (which is very slow), requires some form of screen streaming, or is a paid service.

I believe collaboration is extremely important to both open source and programmers in general. Solutions like Git are great (and Multihack is *no* replacement for them), but for working together at a granular level, line by line, you need something more.

What the community needs is an open, free product that allows reliable, fast and easy-to-use remote pair programming. No one appears to be working on this, so I started. I made it P2P so that it would be feasible to run public servers (and faster). I'm working to get it CCI compliant so it's extremely reliable. Eventually, you should be able to use any popular editor. I already *hope* that it's easy-to-use, but that's always improving.

I want the core featureset (marked **Beta-Critical**) and extensive testing (thanks to everyone for helping with that!) finished before I start promoting this. Then, when we get a large number of users, I'll have plenty of feedback to improve performance.

After that, I might add some of the less critical features, or open up a dead-simple API to attach plugins for new functionality (kind of like extensions for an extension). Then you can go wild with crazy realtime collab features. That'll be the full release, at which point it'll be stable.

What do you think?

**Comment by tweakimp** [comment]
*"Nothing's wrong, I just believe there can always be improvement."* While this is true most of the time, it can be really hard to find the best solution once you found the second best.
I think I got your view, and I'm ready to help :) 
If you want to, we can do a voice call this weekend to discuss even more.

**Comment by t-mullen** [comment]
Great! Please join the gitter if you want a better way to discuss: https://gitter.im/multihack/multihack

**Comment by tweakimp** [comment]
My comments on the current UI:
![ui](https://cloud.githubusercontent.com/assets/20489521/25545459/3ae8702a-2c5f-11e7-9300-ad1f02674f54.png)

This is what the new logo could look like together with the name.
![logo](https://cloud.githubusercontent.com/assets/20489521/25545458/3ae54350-2c5f-11e7-92eb-69ec0318f019.png)

**Comment by tweakimp** [comment]
Here are three logo icon + name combinations. I also tested out some fonts for the new name.
Dont forget it might look different with the white github background.
What do you think?
![logo](https://cloud.githubusercontent.com/assets/20489521/25553534/43892f24-2cb6-11e7-9c5f-a391a3f65974.png)

**Comment by t-mullen** [comment]
I like the second font. Can the logo be smaller so that it is no bigger than the text?

What font is that?

**Comment by tweakimp** [comment]
That would make the logo look a bit small in my opinion.
![logo](https://cloud.githubusercontent.com/assets/20489521/25553572/39267eaa-2cb7-11e7-916e-58615261914f.png)

**Comment by t-mullen** [comment]
You're right. That's too small. Somewhere between then. I'll play around with it when I put the new design in.

**Comment by tweakimp** [comment]
Ok.
I think a good rule of thumb would be to make the logo so big so that  the arrows are half the thinkness of the letters.
The second font is `Source Code Pro`, the Brackets Code font ;)

**Comment by tweakimp** [comment]
Next try, icon is a bit smaller. Two new fonts.
![logo](https://cloud.githubusercontent.com/assets/20489521/25553726/e03538d6-2cbb-11e7-8064-366b6d0c9abc.png)
